apps/src/blockly/addons/cdoFieldDropdown.js

```diff
@@ -1,31 +1,9 @@
 import GoogleBlockly from 'blockly/core';
-import i18n from '@cdo/locale';
+import _ from 'lodash';
 
 const EMPTY_OPTION = '???';
 
 export default class CdoFieldDropdown extends GoogleBlockly.FieldDropdown {
-  static ARROW_CHAR = '\u25BC';
-  static NO_OPTIONS_MESSAGE = i18n.uninitialized();
```

Comment:
We use `FieldDropdown.NO_OPTIONS_MESSAGE` in a couple places in a Sprite Lab blocks.js file.

Reply:
Thanks! What do you think about moving `NO_OPTIONS_MESSAGE` to `blockly/constants.js`?

Reply:
That would probably work fine. It's currently where it is because it's defined in the same place in our fork: https://github.com/code-dot-org/blockly/blob/v4.0.14/core/ui/fields/field_dropdown.js#L86
This value is referenced within the fork a few times.

Reply:
I moved this constant back to this file because I was unable to use an `i18n` string in `blockly/constants.js`. I got the following error:
```
HeadlessChrome 0.0.0 (Mac OS X 10.15.7) ERROR
  Uncaught TypeError: _locale.default.teachingTip is not a function
  at test/unit-tests.js:1349948
```